File: GTTS/INL.VettingService.Models/GetPersonsVettingHit_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class GetPersonsVettingHit_Item : IGetPersonsVettingHit_Item
    {
        public long VettingHitID { get; set; }
        public long PersonsVettingID { get; set; }
        public int VettingTypeID { get; set; }
        public string FirstMiddleNames { get; set; }
        public string LastNames { get; set; }
        public int? DOBYear { get; set; }
        public int? DOBMonth { get; set; }
        public int? DOBDay { get; set; }
        public string PlaceOfBirth { get; set; }
        public byte? ReferenceSiteID { get; set; }
        public byte? HitMonth { get; set; }
        public byte? HitDay { get; set; }
        public int? HitYear { get; set; }
        public long? UnitID { get; set; }
        public string HitUnit { get; set; }
        public string HitLocation { get; set; }
        public byte? ViolationTypeID { get; set; }
        public byte? CredibilityLevelID { get; set; }
        public string HitDetails { get; set; } 
        public string Notes { get; set; }
        public string TrackingID { get; set; }
        public byte? HitResultID { get; set; }
        public string HitResultDetails { get; set; }
        public DateTime? VettingHitDate { get; set; }
        public string First { get; set; }
        public string Middle { get; set; }
        public string Last { get; set; }
        public bool IsRemoved { get; set; }
        public bool isHistorical { get; set; }
        public List<AttachDocumentToVettingHit_Result> VettingHitFileAttachmentJSON { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-persons-vetting-hit_item.ts


import { AttachDocumentToVettingHit_Result } from './attach-document-to-vetting-hit_result';
import { IGetPersonsVettingHit_Item } from './iget-persons-vetting-hit_item';

export class GetPersonsVettingHit_Item implements IGetPersonsVettingHit_Item {
  
	public VettingHitID: number = 0;
	public PersonsVettingID: number = 0;
	public VettingTypeID: number = 0;
	public FirstMiddleNames: string = "";
	public LastNames: string = "";
	public DOBYear?: number;
	public DOBMonth?: number;
	public DOBDay?: number;
	public PlaceOfBirth: string = "";
	public ReferenceSiteID?: number;
	public HitMonth?: number;
	public HitDay?: number;
	public HitYear?: number;
	public UnitID?: number;
	public HitUnit: string = "";
	public HitLocation: string = "";
	public ViolationTypeID?: number;
	public CredibilityLevelID?: number;
	public HitDetails: string = "";
	public Notes: string = "";
	public TrackingID: string = "";
	public HitResultID?: number;
	public HitResultDetails: string = "";
	public VettingHitDate?: Date;
	public First: string = "";
	public Middle: string = "";
	public Last: string = "";
	public IsRemoved: boolean = false;
	public isHistorical: boolean = false;
	public VettingHitFileAttachmentJSON?: AttachDocumentToVettingHit_Result[];
  
}


